scripts/bx_auth.sh

```diff
@@ -3,7 +3,7 @@
 BLUEMIX_ORG="Developer Advocacy"
 BLUEMIX_SPACE="dev"
 
-if [[ -z "$BLUEMIX_AUTH" ]]; then
+if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
     echo -e "\033[0;33mFork detected; not authenticating to Bluemix.\033[0m"
```

Comment:
Keep in mind that this is not actually detecting a fork, it is detecting a pull request. Let's update the message to reflect this.